src/peergos/shared/io/ipfs/api/Multipart.java

```diff
@@ -48,13 +48,38 @@ public void addFormField(String name, String value) {
         writer.flush();
     }
 
+    public void addSubtree(String path, File dir) throws IOException {
```

Comment:
Maybe add  a  comment since this call is  recusive?  Something like ``Adding dirPath + dir.getName()``?